e2e/cypress/integration/enterprise/teams/search_teams_spec.js

```diff
@@ -22,68 +22,55 @@ describe('Search teams', () => {
     beforeEach(() => {
         cy.apiAdminLogin();
         cy.visit('/admin_console/user_management/teams');
-        cy.wrap([]).as('createdTeamIDs');
-    });
-
-    afterEach(function() {
-        this.createdTeamIDs.forEach((teamID) => {
-            cy.apiDeleteTeam(teamID);
-        });
     });
 
     it('loads with no search text', () => {
         // * Check the search input is empty.
-        cy.findByTestId('search-input').should('be.visible').and('have.text', '');
+        cy.findByPlaceholderText('Search').should('be.visible').and('have.text', '');
     });
 
-    it('returns results', function() {
+    it('returns results', () => {
         const displayName = uuidv4();
 
         // # Create a new team.
-        cy.apiCreateTeam('team-search', displayName).then(({team}) => {
-            this.createdTeamIDs.push(team.id);
-        });
+        cy.apiCreateTeam('team-search', displayName);
 
         // # Search for the new team.
-        cy.findByTestId('search-input').type(displayName + '{enter}');
+        cy.findByPlaceholderText('Search').type(displayName + '{enter}');
 
         // * Check that the search results contain the team.
         cy.findAllByTestId('team-display-name').contains(displayName);
     });
 
-    it('results are paginated', function() {
+    it('results are paginated', () => {
         const displayName = uuidv4();
 
         // # Create enough new teams with common name prefixes to get multiple pages of search results.
-        for (let i = 0; i < PAGE_SIZE + 2; i++) {
-            cy.apiCreateTeam('team-search-paged-' + i, displayName + ' ' + i).then(({team}) => {
-                this.createdTeamIDs.push(team.id);
-            });
-        }
+        Cypress._.times(PAGE_SIZE + 2, (i) => {
+            cy.apiCreateTeam('team-search-paged-' + i, displayName + ' ' + i);
+        });
```

Comment:
The for-loop randomly generate an error. The lodash wrapper by Cypress seems to work properly.